src/Response/Entity/Cookie.php

```diff
@@ -0,0 +1,94 @@
+<?php
+declare(strict_types=1);
+
+namespace YapepBase\Response\Entity;
+
+class Cookie
+{
+    /** @var string */
+    protected $name;
+
+    /** @var string */
+    protected $value;
+
+    /** @var int */
+    protected $ttlInSeconds;
+
+    /** @var string */
+    protected $path;
+
+    /** @var string */
+    protected $domain;
+
+    /** @var bool */
+    protected $secure;
+
+    /** @var bool */
+    protected $httpOnly;
+
+    public function __construct(
+        string $name,
+        string $value,
+        int $ttlInSeconds = 0,
+        string $path = '/',
+        string $domain = '',
+        bool $secure = false,
```

Comment:
It would be nice to make this and the $httpOnly default to NULL, and then if NULL they would have a configurable default.

So say you have a dev env, there you don't use https because you don't want to mess with the cert, so you don't use secure cookies. But on prod you want to have https, and secure your cookies so they can't be sent over http. It would be nice if the default for this could come from the config.

Probably the same applies to the domain too and possibly the path too.

Reply:
The idea is good. I'm completely agains using the Config though. So I created a simple factory what can be set to the DI and that factory will handle the secure and https flags the same way.